aries_cloudagent/config/default_context.py

```diff
@@ -120,6 +121,7 @@ async def load_plugins(self, context: InjectionContext):
         # Register external plugins
         for plugin_path in self.settings.get("external_plugins", []):
             plugin_registry.register_plugin(plugin_path)
+        plugin_registry.register_plugin("aries_cloudagent.resolver")
```

Comment:
I think this should be placed with the other admin route registrations for builtin stuff, perhaps around line 116 or so.